downstream/modules/platform/proc-controller-inv-source-terraform.adoc

```diff
@@ -0,0 +1,40 @@
+[id="proc-controller-inv-source-terraform"]
+
+// This Terraform module is for AAP 2.5
+
+= Terraform State
+
+Configure an inventory sourced from a Terraform State file.
+
+.Procedure
+
+. From the navigation panel, select {MenuAEProjects}.
+. On the *Projects* page, click btn:[Create project] to start the *Create Project* window.
+** Enter the appropriate details according to the steps in xref:proc-controller-adding-a-project[Adding a new project].
+** Use the configuration files at link:https://github.com/hakbailey/terraform-aap-demo[terraform-aap-demo].
```

Comment:
We probably don't want to link to my personal repo here. Which configuration files are being referenced?

Reply:
Removed this sub-step.